templates/user/privacy.html

```diff
@@ -1,41 +1,112 @@
 {% extends "layout.html" %}
 
+{% block extra_css %}
+<link rel=stylesheet href={{ url_for('static', filename='css/privacy.css') }}>
+{% endblock %}
+
 {% block main %}
-<style>
-table {
-  margin: 0 auto;
-  border-collapse: collapse;
-}
-tr:not(:first-child,:last-child) {
-  border-bottom: 1px solid #ccc;
-}
-td {
-  padding: 1rem;
-}
-.delete {
-  background-color: #fee7e7;
-  color: #a24343;
-  border-color: #dab2b2;
+<section>
+  <article>
+    <h2>Community mode</h2>
+    <form>
+      <input type=checkbox name=community {% if user.allow_community %}checked{% endif %} id=community>
+      <label for=community title="{% if user.allow_community %}Disable{% else %}Enable{% endif %} community profile">
+        Click to {% if user.allow_community %}disable{% else %}enable{% endif %}
+      </label>
+    </form>
+    <aside>
+      {% if user.allow_community %}
+      <p>
+        Disable community mode hidden your public data as publication, localisation and account.
+      </p>
+      {% else %}
+      <p>
+        Enable community mode allow you to share your articles, localisation and account.
+      </p>
+      {% endif %}
+    </aside>
+    {% if user.allow_community %}
+    <footer>
+      <p>
+        Even if you disable the community mode you can still read, write and use your maps.
+      </p>
+    </footer>
+    {% else %}
+    {% endif %}
+  </article>
+  <article>
+    <h2>List of your trip positions</h2>
+    <ul>
+      {% for location in locations %}
+      <li>
+        <p>{{ location.date | datetime }}</p>
+        <a 
+           data-latlng={{ location.position|join(',') }}  
+           href=https://osm.org/?mlat={{ location.position[0] }}&mlon={{ location.position[1] }} 
+           target=_blank>{{ location.position|join(',') }}
+        </a>
+        <form method=post action={{ url_for('privacy_delete_trip', date=location.date.timestamp()) }}>  
+          <a class=button href=https://osm.org/?mlat={{ location.position[0] }}&mlon={{ location.position[1] }} target=_blank>View on map</a>
+          <button class=delete>Delete</button>
+        </form>
+      </li>
+      {% endfor %}
+    </ul>   
+  </article>
+</section>
+{% endblock %}
+
+{% block extra_js %}
+<script>
+const communityField = document.querySelector('[name=community]')
+communityField.addEventListener('change', event => {
+  call = api.send('patch', "{{ url_for('profile_edit') }}",
+    {allow_community: event.target.checked},
+  )
+  call = api.send('patch', "{{ url_for('profile_edit') }}",
+    {allow_localization: event.target.checked},
+  )
+  if (event.target.checked) {
+    apiFeedback(call, 'You enable community mode, you share your informations, authorized contents and position.')
+    localize()
+  } else {
+    apiFeedback(call, 'Your disable community mode, your hidden your profile and datas.')
```

Comment:
Community mode was disabled. Your profile, location and articles are now hidden.